_posts/2019-03-12-so-you-want-to-be-an-engineer.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: epic
+title: "So You Want to Be an Engineer"
+date: "2019-03-12"
+author: [matt_dole]
+categories: [artsy, beginners, culture, engineering, people, team]
+---
+
+First of all, that's very exciting! Software engineering is pretty darn cool—you get to learn lots of new things,
+understand the technology you use every day better, and contribute to the mysterious maw known as "the internet".
+
+Last February, I also decided that I wanted to pursue computer engineering. I'd been at Artsy for a bit less than
+two years at that point, first as a marketing intern working on SEO and then as a coordinator on the CRM (read:
+email) team. I'd consistently been working on small technical projects; first doing
+[some work](https://github.com/artsy/positron/commit/3176282a3ea94c626e9d851b7c0dd27a1bb0fcb4) on a tool for SEO
```

Comment:
Amazing that we can link to the actual commit – OSS ftw!

Reply:
lol yep I was like "I wonder if I can find that commit I made 3 years ago" and it was surprisingly easy